sklearn/utils/validation.py

```diff
@@ -155,6 +156,38 @@ def _shape_repr(shape):
     return "(%s)" % joined
 
 
+def check_memory(memory):
+    """Check that the ``memory`` is Memory-like.
```

Comment:
Remove the "the" and use joblib.Memory-like for consistency: "Check that ``memory`` is joblib.Memory-like"